src/test/java/de/caritas/cob/userservice/api/manager/consultingtype/ConsultingTypeManagerTest.java

```diff
@@ -72,8 +73,17 @@ void isConsultantBoundedToAgency_Should_Return_False()
   @Test
   void getAllConsultingTypeIds_Should_Return_The_Same_List()
```

Comment:
please also add a test for non-mulititenant context to make sure, this will always work, even if we disable feature toggle

Reply:
getAllConsultingTypeIds_Should_Return_The_Same_List -> this include also a test without no tenant context. first assert is without tenant context

assertEquals(consultingTypeManager.getAllConsultingTypeIds(), List.of(1, 2, 3, 4, 5, 6, 7, 8));
    TenantContext.setCurrentTenant(tenant1);
    assertEquals(consultingTypeManager.getAllConsultingTypeIds(), List.of(1, 2, 3, 4));
    TenantContext.setCurrentTenant(tenant2);
    assertEquals(consultingTypeManager.getAllConsultingTypeIds(), List.of(5, 6, 7, 8));